widgetastic_manageiq/__init__.py

```diff
@@ -1678,6 +1678,79 @@ def read(self):
         return self.selected
 
 
+class CompareToolBarMixin(View):
+    """ It represent abstract class for compare toolbar actions. """
+
+    @property
+    def _view_buttons(self):
+        """It should be override as per concern nested class buttons"""
+        pass
+
+    def select(self, title):
+        for button in self._view_buttons:
+            if button.title == title:
+                return button.click()
+        else:
+            raise ValueError("The Mode with title {title} isn't present".format(title=title))
+
+    @property
+    def selected(self):
+        if self.is_displayed:
+            return next(btn.title for btn in self._view_buttons if btn.active)
+        else:
+            return None
+
+    def read(self):
+        return self.selected
+
+
+class CompareToolBarViewActions(View):
```

Comment:
if you take a look at our code, you will notice that each view name has View in the end. So, your view should have name like CompareToolBarActionsView.